src/styles/blocks/container.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  position: relative;
+  top: 50%;
+  transform: translateY(50%);
+  width: 500px;
+  height: 500px;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.